kemeny_sourcemash.tex

```diff
@@ -35,7 +35,9 @@
 \pagebreak
 \section{Project Overview}
 
-Sourcemash 
+Sourcemash is a web-based application that categorizes news articles to help users read their RSS feeds faster. Sourcemash achieves this goal is three steps. First, users select their sources from the Browse page, which offers a pre-seeded list of popular feeds and a form allowing manual input of new RSS feeds; Sourcemash is designed to categorize your news, not someone else's. Second, Sourcemash's categorization engine analyzes each article in the user's subscribed feeds to find the categories that best match; our algorithm is constantly working in the background, so users can find articles in the topics they're looking for faster. Finally, Sourcemash offers the ability to read thew news by both category and feed, offering users the ability to integrate their news reading preferences into the application experience. 
```

Comment:
How's:

Sourcemash is a web-based RSS feed aggregator that algorithmically categorizes articles for users to browse their feed by category. Typically, RSS feed readers only allow users to look through their news feed-by-feed. By categorizing articles and regrouping them in this way, users can find the topics that interest them faster and without re-reading similar stories from different sources. Sourcemash is ideal for news junkies who want the full picture on a story in a way that's easier to digest and consumes less of their time.